archivebox/themes/default/link_details.html

```diff
@@ -233,7 +233,7 @@
                 }
             }
         </style>
-        <link rel="stylesheet" href="../../static/bootstrap.min.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css">
```

Comment:
can you change this back, it has to all be self-hosted so that it can operate 100% offline

Reply:
@pirate I was going to ask about this. The js is actually using the cdn, so I will need to change that too. The static indexes (outside of django) cannot find these files. Does the static folder needs to be copied to the archive and I broke it? I am almost done fixing the tests (just one missing). I am starting to check against the current live version.